### PR TITLE
Add techtree method in init, that return a dictionary with technology…

### PR DESCRIPTION
**Pull request by Systemac**
…, supply and facility necessary to obtain it.

Sorry for bad camelcase and variable'names...

### CODE REVIEW

File: ogame/__init__.py [hunk 1]
```diff
@@ -319,6 +341,10 @@ class Resources:
             day_production = [int(day_production[0].span['title'].replace('.','')),
                               int(day_production[1].span['title'].replace('.','')),
                               int(day_production[2].span['title'].replace('.','')),]
+            storage = bs4.find_all('tr', attrs={'class':'alt'})[7].find_all('td', attrs={'class':'normalmark left2'})
```
Comment (PiecePaperCode) [inline ogame/__init__.py]: i like that

File: ogame/__init__.py [hunk 2]
```diff
@@ -298,6 +298,28 @@ def celestial_coordinates(self, id):
                 coordinates.append(const.destination.moon)
                 return coordinates
 
+    def techtree(self, id):
```
Comment (PiecePaperCode) [inline ogame/__init__.py]: as mention in Discord we can save it offline and save the result in constnts.py

Comment (Systemac) [reply ogame/__init__.py]: yes, but in another class, i don't want to break your code ^^

**Comment by Systemac** [comment]
Done, if it's good for you...

**Comment by Systemac** [comment]
End of bugfixes and abandon of treetech...

**Comment by Systemac** [comment]
End of bugfixes and abandon of treetech...

**Comment by PiecePaperCode** [comment]
thx. for your work. im starting to i tegrate your techtree into the lib

**Comment by Systemac** [comment]
> thx. for your work. im starting to i tegrate your techtree into the lib

Thx for you ^^

**Comment by PiecePaperCode** [comment]
# Tests are all passing
```py
test_Constants (ogame.test.UnittestOgame) ... ok
test_Events (ogame.test.UnittestOgame) ... ok
test_Vars (ogame.test.UnittestOgame) ... ok
test_ally (ogame.test.UnittestOgame) ... ok
test_build (ogame.test.UnittestOgame) ... ok
test_celestial (ogame.test.UnittestOgame) ... ok
test_celestial_coordinates (ogame.test.UnittestOgame) ... ok
test_defences (ogame.test.UnittestOgame) ... ok
test_facilities (ogame.test.UnittestOgame) ... ok
test_fleet (ogame.test.UnittestOgame) ... ok
test_galaxy (ogame.test.UnittestOgame) ... ok
test_moon_facilities (ogame.test.UnittestOgame) ... ok
test_phalanx (ogame.test.UnittestOgame) ... ok
test_research (ogame.test.UnittestOgame) ... ok
test_resources (ogame.test.UnittestOgame) ... ok
test_send_fleet (ogame.test.UnittestOgame) ... ok
test_send_message (ogame.test.UnittestOgame) ... ok
test_ships (ogame.test.UnittestOgame) ... ok
test_spyreports (ogame.test.UnittestOgame) ... ok
test_supply (ogame.test.UnittestOgame) ... ok

----------------------------------------------------------------------
Ran 20 tests in 35.679s

OK

Process finished with exit code 0
```

**Pull request merged**